docs/navigation.md

```diff
@@ -3,72 +3,26 @@ id: navigation
 title: Navigating Between Screens
 ---
 
-Mobile apps are rarely made up of a single screen. Managing the presentation of, and transition between, multiple screens is typically handled by what is known as a navigator.
+By default, React Native renders your components onto a single screen. To create multiple screens and navigate between them you need a navigation library. The following table helps you choose the one that's right for you. Each column identifies a different feature and each row represents a different navigation library (React Router Native isn't included because it doesn't have stack navigation). These features aren't the be-all and end-all of what makes a good navigation library but they are important. Keep reading to find out why.
 
-This guide covers the various navigation components available in React Native. If you are just getting started with navigation, you will probably want to use [React Navigation](navigation.md#react-navigation). React Navigation provides an easy to use navigation solution, with the ability to present common stack navigation and tabbed navigation patterns on both iOS and Android.
+|                                                                                                           | Uses native API | Has one React root | Runs in Expo | Runs on the Web |
+| --------------------------------------------------------------------------------------------------------- | --------------- | ------------------ | ------------ | --------------- |
+| [React Navigation](https://reactnavigation.org/docs/en/getting-started.html)                              | No              | Yes                | Yes          | No              |
+| [React Native Navigation](https://wix.github.io/react-native-navigation/)                                 | Yes             | No                 | No           | No              |
+| [The Navigation router](https://grahammendick.github.io/navigation/documentation/native/hello-world.html) | Yes             | Yes                | No           | Yes             |
 
-If you'd like to achieve a native look and feel on both iOS and Android, or you're integrating React Native into an app that already manages navigation natively, the following library provides native navigation on both platforms: [react-native-navigation](https://github.com/wix/react-native-navigation).
+## Uses native API
 
-## React Navigation
+React Native uses the same fundamental UI building blocks as regular apps. Both iOS and Android have fundamental navigation building blocks Android has `Activities` or `Fragments` and iOS has `UIViewControllers`. If your navigation library doesn't use them then you can end up with an app that doesn't have a native look and feel. You'll also miss out on iOS navigation bar features like large titles and search bars.
```

Comment:
> If your navigation library doesn't use them then you can end up with an app that doesn't have a native look and feel.

This doesn't seem accurate, if you don't use the platform features you can still have a native look and feel if you emulate the platform features. This section should include the advantages of not using the platform if it want's to be unbiased.

Reply:
> This doesn't seem accurate

But it is. I love `react-navigation` but the emulated IOS and Android navigation UI is merely an approximation or the real thing. It’s “good enough”, but it’s not perfect and you can really feel it.

I think we can tweak the wording to reflect it — it’s not that non-Native implementations suck, but it is a real tradeoff

Reply:
I don't think this tradeoff is binary, it's not native or not native. For example, we used to use DrawerLayoutAndroid for the drawer navigator implementation, which I suppose you might call native navigation then. However, we found that it was buggy and not customizable enough so we re-implemented it with react-native-gesture-handler and react-native-reanimated. Going in the other direction, @kmagiera has explored building a set of primitives that navigation libraries build upon with react-native-screens. When he first prototyped it, he made a native stack that you could build react-navigation on top of. He has since returned to that and is exploring a backend for the React Navigation Stack Navigator that uses UINavigationController behind the scenes. The result, if it works out well, could be that you can have a fully customizable RNGH/Reanimated powered implementation and one that is driven by native primitives (although limited to a subset of the options).

Reply:
The particular sentence here is also incorrect because you can opt-in to using react-native-screens with React Navigation by just doing `import { useScreens } from 'react-native-screens'; useScreens()` and your app will use [UIViewControllers](https://github.com/kmagiera/react-native-screens/blob/4dd751811addf7493bd355e5d557270c086979ed/ios/RNSScreen.m) and [Fragment]s(https://github.com/kmagiera/react-native-screens/blob/ed826d7b91acd84ced6fe933db74281fed469d69/android/src/main/java/com/swmansion/rnscreens/ScreenContainer.java) to represent screens.